samples/BasicEventSourcingSample/Core/ShipApplys.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using CleanArchitecture.Exceptions;
+using Microsoft.Azure.CosmosEventSourcing;
+
+namespace BasicEventSourcingSample.Core;
+
+public partial class Ship
```

Comment:
I renamed this file to _Ship.Apply.cs_, singular and use the `.` to clearly separate the `Ship` and it's partial implementations specific intent.

Reply:
Yeah, I like this. It makes sense.